js/index.js

```diff
@@ -1,6 +1,36 @@
-// Iteration 1: Names and Input
+// console.log("I'm ready!");
+let hacker1 = "Helena";
+console.log("The driver's name is " + hacker1);
 
-// Iteration 2: Conditionals
+let hacker2 = "Denise";
+console.log("The navigator's name is " + hacker2);
 
 
-// Iteration 3: Loops
+// iteration 2: conditionals
+if (hacker1.length > hacker2.length) {
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters`); 
+} else if (hacker1.length < hacker2.length) {
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters`);  
+} else if (hacker1.length == hacker2.length) {
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`);
+}
+
+// iteration 3: loops
+//3.1
+let spaces= hacker1.toUpperCase(); 
+console.log(` ${spaces.charAt(0)} ${spaces.charAt(1)} ${spaces.charAt(2)} ${spaces.charAt(3)} ${spaces.charAt(4)} ${spaces.charAt(5)} `);
+//3.2
```

Comment:
That's what you and Luis were talking about this morning. So no problem that you got it wrong and great that you didn't just copy the other solution.